crates/rustc_codegen_spirv/src/codegen_cx/entry.rs

```diff
@@ -179,7 +179,8 @@ impl<'tcx> CodegenCx<'tcx> {
             SpirvType::Image { .. }
             | SpirvType::Sampler
             | SpirvType::SampledImage { .. }
-            | SpirvType::AccelerationStructureKhr => {
+            | SpirvType::AccelerationStructureKhr
+            | SpirvType::RayQueryKhr => {
```

Comment:
I don't think it should be added here, since it's invalid to have in the interface.